frontend/src/lib/components/flows/FlowEditor.svelte

```diff
@@ -22,11 +21,9 @@
 					/>
 				{/if}
 			</div>
-		</left>
-		<right slot="right" class="h-full">
-			<div class="h-full overflow-auto bg-white ">
-				<FlowEditorPanel {initialPath} />
-			</div>
-		</right>
-	</HSplitPane>
+		</Pane>
+		<Pane size={75} minSize={40} class="!overflow-hidden">
```

Comment:
The class `!overflow-hidden`hides the fact that the Pane component is not sized properly:

https://user-images.githubusercontent.com/456655/196009221-0667ff99-a003-4104-92f8-bf35829e1133.mov